etl/steps/export/explorers/climate/latest/climate_change.config.yml

```diff
@@ -0,0 +1,337 @@
+definitions:
+  common_views:
+    - config:
+        relatedQuestionUrl: https://ourworldindata.org/explorers/climate-change
+        relatedQuestionText: "Data explorer: how is the Earth's climate changing?"
+
+config:
+  explorerTitle: Climate Change Impacts
+  isPublished: true
+  entityType: region
+  explorerSubtitle: Explore the impacts of global climate change.
+  selection:
+    - World
+    - Antarctica
+    - Gulkana Glacier
+    - Lemon Creek Glacier
+    - North America
+    - South Cascade Glacier
+    - Wolverine Glacier
+    - Hawaii
+    - Arctic Ocean
+  subNavId: co2
+  subNavCurrentId: climate-impacts
+  wpBlockId: '45009'
+  hasMapTab: false
+  hideAnnotationFieldsInTitle: true
+
+dimensions:
+  - slug: metric
+    name: Metric
+    choices:
+      - slug: temperature_anomaly
+        name: Temperature anomaly
+      - slug: sea_surface_temperature_anomaly
+        name: Sea surface temperature anomaly
+      - slug: ocean_heat_content__top_700m
+        name: Ocean heat content (top 700m)
+      - slug: ocean_heat_content__top_2000m
+        name: Ocean heat content (top 2000m)
+      - slug: ocean_ph
+        name: Ocean pH
+      - slug: sea_level_rise
+        name: Sea level rise
+      - slug: arctic_sea_ice_extent
+        name: Arctic sea ice extent
+      - slug: antarctic_sea_ice_extent
+        name: Antarctic sea ice extent
+      - slug: mass_balance_of_ice_sheets
+        name: Mass balance of ice sheets
+      - slug: mass_balance_of_u_s__glaciers
+        name: Mass balance of U.S. Glaciers
+      - slug: snow_cover
+        name: Snow cover
+      - slug: co2_concentrations
+        name: CO₂ concentrations
+      - slug: ch4_concentrations
+        name: CH₄ concentrations
+      - slug: n2o_concentrations
+        name: N₂O concentrations
+    presentation:
+      type: dropdown
+  - slug: long_run_series
+    name: Long-run series
+    choices:
+      - slug: false
+        name: 'false'
+      - slug: true
+        name: 'true'
+    presentation:
+      type: checkbox
+views:
+  - dimensions:
+      metric: temperature_anomaly
+      long_run_series: false
+    indicators:
+      y:
+        - catalogPath: grapher/met_office_hadley_centre/2025-01-21/near_surface_temperature/near_surface_temperature#temperature_anomaly
```

Comment:
maybe you can consider removing the `grapher/met_office_hadley_centre/2025-01-21/near_surface_temperature/` bit for all indicators? Could help automate future updates.